doc/source/gitwash/development_workflow.rst

```diff
@@ -121,7 +121,7 @@ In more detail
      # On branch ny-new-feature
      # Changed but not updated:
      #   (use "git add <file>..." to update what will be committed)
-     #   (use "git checkout -- <file>..." to discard changes in working directory)
+     #   (use "git switch -- <file>..." to discard changes in working directory)
```

Comment:
This change is incorrect.

Reply:
Oh, I am sorry. @stefanv  Should I use "git switch --detach <commit>" instead?